src/Table/index.js

```diff
@@ -0,0 +1,245 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import SortBy from 'lodash/sortBy';
+import Reverse from 'lodash/reverse';
+
+import { Icon, Theme } from '..';
+import {
+  Container,
+  Header,
+  HeaderRow,
+  HeaderCell,
+  Body,
+  BodyRow,
+  BodyCell,
+  HeaderCellTitle,
+  HeaderSortingContainer,
+  HeaderSortingGhost,
+  HeaderLabel,
+} from './elements';
+
+/** Lookup object for next sorting direction. */
+const nextSortingDirection = {
+  none: 'asc',
+  asc: 'desc',
+  desc: 'none',
+};
+
+/** Lookup object for icon name from sorting direction. */
+const sortingDirectionToIconName = {
+  none: 'carets-vertical',
+  asc: 'caret-up',
+  desc: 'caret-down',
+};
+
+/**
+ * Defines a Table component.
+ *
+ * Displays given data into a table layout and manages the data.
+ */
+class Table extends PureComponent {
+  /** Prop types. */
+  static propTypes = {
```

Comment:
For proptypes you should destructure like `const { node, func, .... } = PropTypes`, it will be more clear